functional-samples/sample.co2meter/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "CO2 meter extension",
-  "description": "An extension that connects to a CO2  meter using WebHID extension service workers",
+  "description": "An extension that connects to a CO2 meter using WebHID extension service workers",
```

Comment:
```suggestion
  "description": "Demonstrates using WebHID to connect to a CO2 meter.",
```

Reply:
@oliverdunk Is there a way to hide this until WebHID is actually released?

Reply:
@daidr, is that something you could look at? My initial thought is that a list of exclusions in the developer.chrome.com code probably makes sense, but I could be convinced it should live elsewhere :)

Reply:
Addressed here: https://github.com/GoogleChrome/developer.chrome.com/commit/4480490df6e95ea6dc1ab304180c73940cb5fff6